test/nbrowser/DocUsageTracking.ts

```diff
@@ -0,0 +1,179 @@
+import {UserAPI} from 'app/common/UserAPI';
+import {assert, driver, Key} from 'mocha-webdriver';
+import fetch from 'node-fetch';
+import * as gu from 'test/nbrowser/gristUtils';
+import {server} from 'test/nbrowser/testServer';
+import {setupTestSuite} from 'test/nbrowser/testUtils';
+import {
+  enableExternalAttachmentsForTestSuite
+} from 'test/nbrowser/externalAttachmentsHelpers';
+
+describe('DocumentUsage', function() {
```

Comment:
```suggestion
describe('DocUsageTracking', function() {
```

The convention is to have the same name as the file.